lib/components-schema-v1_7_x.ts

```diff
@@ -35,6 +35,48 @@ function labelProperty(description: string): { oneOf: JSONSchema7Definition[] }
     };
 }
 
+const inlineComponentPropertyDefinitionOrReferenceList: { oneOf: JSONSchema7Definition[] } = {
+    oneOf: [
+        {
+            type: 'string',
+        },
+        {
+            type: 'object',
+            additionalProperties: false,
+            required: ['name', 'directiveKey'],
```

Comment:
https://github.com/WoodWing/csde-components-boilerplate/blob/master/components/components-definition.json#L34

So we can also allow:
`{ "name": "prop", "defaultValue": "value" }`

Reply:
Added.